napari/components/viewer_model.py

```diff
@@ -1587,12 +1587,12 @@ def _unify_data_and_user_kwargs(
 ) -> FullLayerData:
     """Merge data returned from plugins with options specified by user.
 
-    If ``data == (_data, _meta, _type)``.  Then:
+    If ``data == (data_, meta_, type_)``.  Then:
```

Comment:
I assume putting the underscore at the end is to appease some formatting rule?

Reply:
The `_name` is pattern for unused named variable.